client/src/components/resources/configs/cellLine.js

```diff
@@ -0,0 +1,65 @@
+export const searchResult = ['tissue_histology', 'available_quality_fields']
+
+export const details = [
+  {
+    basic_information: [
+      'cell_line_name',
+      'organism',
+      'tissue',
+      'disease',
+      'number_of_availible_cell_lines',
+      'age',
+      'sex',
+      'ethnicity',
+      'bio_safety_level',
+      'population_doubling_time'
+    ]
+  },
+  {
+    culture: [
+      'storage_medium',
+      'growth_medium',
+      'subculturing',
+      'cryopreservation',
+      'culture_conditions'
+    ]
+  },
+  {
+    quality: ['str_profile', 'passage_number', 'description']
+  }
+]
+
+export const form = [
```

Comment:
You know, for the ones where form and details are exactly identical can you just copy the details object to make that obvious. If they diverge in the future the details can just be copied and pasted then modified.

Reply:
yeah i think thats a good idea i can create an issue to simplify that stuff